arkouda/pdarrayclass.py

```diff
@@ -1171,6 +1191,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list:
```

Comment:
I suggest replacing this with List[str]

Reply:
@hokiegeek2, I think this should be updated now. I changed the return type to `list[str]`. I'm not sure if the capitalization was important here but mypy complained less with this way

Reply:
@pierce314159 Gothca, yeah, unfortunately, I don't believe list[str] works for 3.7:

![image](https://user-images.githubusercontent.com/10785153/114874303-34386d80-9dca-11eb-87ae-c1b600f986b2.png)

So pretty sure you need to switch to List[str]